tests/test_cacerts_from_env.py

```diff
@@ -0,0 +1,20 @@
+import os
+import httplib2
+
+try:
+    import certifi
```

Comment:
Test requirements don't specify `certifi` explicitly, but it may be installed as dependency of another packet in future. So seemingly unrelated change will affect how CA is tested.

Solution: separate `certifi` import and usage. That allows to mock presence of certifi in test. Otherwise to test with/out certifi you'd have to run subtest in another Python process or patch/reload `sys.modules[httplib2.certs]` or something even less reliable.

```python
# httplib2/certs.py
import os
# import os.path - this is not needed really, previous line does the job, sorry it also slipped in original review

ca_certs_locater = None
try:
  import ca_certs_locater
except ImportError:
  pass

certifi = None
try:
  import certifi
except ImportError:
  pass

BUILTIN_CA_CERTS = os.path.join(...)

def where():
  env = os.environ.get('HTTPLIB2_CA_CERTS')
  if env is not None:
    return env
  if ca_certs_locater:
    return ca_certs_locater.get()
  if certifi:
    return certifi.where()
  return BUILTIN_CA_CERTS
```